app/assets/javascripts/angular/controllers/static_pages.js.erb

```diff
@@ -0,0 +1,57 @@
+app.controller('StaticPagesCtrl', ['$scope', '$http', function($scope, $http){
+
+  $scope.book = {};
+  $scope.form = {};
+
+  $scope.form.create = function(){
+    var url = "<%= ENV['URL'] %>" || "http://localhost:3000";
+
+    var data = {
+      book: $scope.book
+    };
+
+    $http.post(url + '/books', data).success(function(response){
+      console.log(response);
+    });
+  };
+
+  $scope.apiSearch = function(keyEvent, value) {
+    if (keyEvent.which === 13) {
+      var url = 'https://www.googleapis.com/books/v1/volumes?q=' + value.replace(' ', '+')
+      $http.get(url).success(function(response){
+        $scope.books = response.items
+        console.log($scope.books)
+
+
+        $scope.books.forEach(function(book){
```

Comment:
The api response with an array of 10 books. I am trying to loop through all books and modify their keys to fit what I will need when sending a POST request to the backend.

book = book.volumeInfo is a good idea!!
however, some keys dont match up, any ideas on how to keep it clean?
e.g. 
require [published_date]  instead of [publishedDate]
require [page_count] instead of [pageCount] etc